tests/src/VerboseMessengerTest.php

```diff
@@ -43,16 +42,10 @@ protected function setUp() {
      * @covers ::addMessage
      */
     public function testAddMessage() {
-      // Test adding a message.
       $this->assertTrue($this->messenger->addMessage("Test message"), "The message was added");
     }
 
-    /**
-     * Tests add messages.
-     * @covers ::addMessage
```

Comment:
I agree I am not familiar properly with Drupal 8 testing standard, BUT there is just one method in the main class which is already covered by the previous test method. 

So, the question is, is it necessary to have that cover thing in all test methods?

IMO both methods in this particular case can be merged in one with two assertions